### DISCUSSION TITLE
Questions to be graphed for the GloBIAS website

### DISCUSSION THREAD

**Discussion by bethac07** [GloBIAS-BioimageAnalysts/Survey_2024#12] [Ideas]
Will make one per comment below, to keep discussion nicely threaded. Can link to issues in each one.

**Comment by bethac07** [comment]
Question 10 about what should GloBIAS do (#8)

**Comment by ChristaGW** [comment]
Question 13 and Question 10/ 11 - I would like to know how many people want (any) event (ansered Q10 and/or Q11) and how many actually answered yes to Q13 (as questions weren't mandatory just looking at yes and no might be less informative).

**Comment by ChristaGW** [comment]
How many of the people saying yes in Q13 actually answered Q17? E.g. not only saying yes but having a clear preference for a topic.
![survey - filter Q13 plus Q17](https://github.com/user-attachments/assets/33f18505-fb88-4ef0-9597-50ef11faa0df)

**Comment by ChristaGW** [comment]
We have nice graphs now for Q15 (reasonable fee). Is there a way to filter this by Q16-  would be paid by institution (assuming that people are less bothered about how much if they don't pay).

**Comment by ChristaGW** [comment]
A summary of Q19 against summary of Q22 - which I could cross check with the actuall number of volunteers we had after the survey.

**Comment by ChristaGW** [comment]
Question 8 - involved with NEUBIAS before: How many of the NO are based in Europe?
